dropwizard-core/src/main/java/io/dropwizard/setup/Environment.java

```diff
@@ -71,6 +78,15 @@ public Environment(String name,
 
         this.jerseyServletContainer = new JerseyContainerHolder(new ServletContainer(jerseyConfig));
         this.jerseyEnvironment = new JerseyEnvironment(jerseyServletContainer, jerseyConfig);
+
+
+        this.healthCheckExecutorService = this.lifecycle().executorService("TimeBoundHealthCheck-pool-%d")
+                .workQueue(new ArrayBlockingQueue<Runnable>(4))
```

Comment:
I think we don't need a queue with a size more than 1. There are no reason in preserving the pool size.
In case if DB is down and 2 concurrent request happen, the situation is actually worse, because the second request
wiil be forced to wait whole 5 seconds before it will be processed.